service/src/main/java/bio/terra/catalog/service/BaseStatusService.java

```diff
@@ -57,16 +58,12 @@ void registerStatusCheck(String name, Supplier<SystemStatusSystems> checkFn) {
   void checkStatus() {
     if (configuration.enabled()) {
       var newStatus = new SystemStatus();
-      try {
-        var systems =
-            statusCheckMap.entrySet().stream()
-                .collect(Collectors.toMap(Map.Entry::getKey, e -> e.getValue().get()));
-        newStatus.setOk(systems.values().stream().allMatch(SystemStatusSystems::isOk));
-        newStatus.setSystems(systems);
-      } catch (Exception e) {
```

Comment:
I think it's incorrect to remove this code. The try/catch is here to handle the case where a subsystem throws an exception when its status check method is called. When a status check throws an exception we want to capture that error and show that the subsystem is down. You can test this case using code like:
```
  void getCurrentStatusException() {
    var config = new StatusCheckConfiguration(true, 0, 0, 10);
    BaseStatusService service = new BaseStatusService(config);
    var status = new SystemStatusSystems().ok(true);
    service.registerStatusCheck("test", () -> {throw new RuntimeException("failure");});
    SystemStatus systemDown = new SystemStatus().ok(false);
    assertThat(service.getCurrentStatus(), is(systemDown));
    service.checkStatus();
    assertThat(service.getCurrentStatus(), is(systemDown));
  }
```